experimental/addons/spark-kubernetes-operator/definitions/spark-application.cue

```diff
@@ -0,0 +1,89 @@
+"spark-application": {
```

Comment:
```suggestion
"spark-workload": {
```

Will it be btter to name as `workload` instead of application? Because it will finally composed as a KubeVela application in yaml.

Reply:
Reasonable. Why I named it to be `spark-application` is that it defines the spec of the spark application.